sdk/identity/identity/review/identity.api.md

```diff
@@ -210,6 +212,63 @@ export class ManagedIdentityCredential implements TokenCredential {
     getToken(scopes: string | string[], options?: GetTokenOptions): Promise<AccessToken>;
     }
 
+// @public
+export enum RegionalAuthority {
+    AsiaEastValue = "eastasia",
```

Comment:
Do we have to have `Value` as a suffix for all of these enum entries? This feels super weird to me.

Reply:
Good question! cc: @schaabs can we remove *Value at the end?

Reply:
It seems the `Value` suffix on these, was copied over by mistake. In the .NET implementation these were copied from the `Value` suffix only applies to the internal constant strings, not the extensible enum values.

Reply:
Oh those were internal .NET fields! My bad! Cleaning this up